src/autogluon/cloud/predictor/timeseries_cloud_predictor.py

```diff
@@ -50,8 +50,8 @@ def fit(
         *,
         predictor_init_args: Dict[str, Any],
         predictor_fit_args: Dict[str, Any],
-        id_column: str = "item_id",
-        timestamp_column: str = "timestamp",
+        id_column: Optional[str] = None,
+        timestamp_column: Optional[str] = None,
```

Comment:
What is our motivation for changing the defaults here?

Reply:
+1

Reply:
I was trying to make the `.predict()` and `.predict_real_time()` API align with what we have in the Chronos [tutorial](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-chronos.html). I see that we might not have `id_column` and `timestamp_column` in the train_data, but please correct me if I misunderstood the example.